examples/v2/vlan_attachment/vlan_attachment.py

```diff
@@ -0,0 +1,87 @@
+def networkUrl(project_id, name):
+	return 'https://www.googleapis.com/compute/v1/projects/%s/global/networks/%s' % (project_id, name)
+
+def interconnectUrl(project_id, name):
+	return 'https://www.googleapis.com/compute/v1/projects/%s/global/interconnects/%s' % (project_id, name)
+
+def GenerateConfig(context):
+	resources = []
+
+	router_name = context.env['deployment'] + '-router'
+	vlan_attach_name = context.env['deployment'] + '-vlan-attach'
+	router_interface_name = router_name + '-if' 
+	bgp_peer_name = router_interface_name + '-peer'
+
+	# Adding Cloud Router
+	resources.append({
+		'name': router_name,
```

Comment:
These are tabbed over pretty far. Could you use a format more like our other samples? Also, I'm not sure how to check on GitHub but please make sure you're using spaces and not tabs.

For example:

https://github.com/GoogleCloudPlatform/deploymentmanager-samples/blob/master/examples/v2/cloud_functions/python/cloud_function.py

Reply:
Fixed